dev/tools/update_icons.dart

```diff
@@ -5,12 +5,15 @@
 // Regenerates the material icons file.
 // See https://github.com/flutter/flutter/wiki/Updating-Material-Design-Fonts-&-Icons
 
+import 'dart:collection';
 import 'dart:convert' show LineSplitter;
 import 'dart:io';
 
 import 'package:args/args.dart';
 import 'package:path/path.dart' as path;
 
+typedef TokenPairMap = Map<String, String>;
```

Comment:
nit: Don't necessarily need to change anything here, but I wonder if `TokenPairMap` is more readable than `Map<String, String>` when defining types. I think `Map<String, String>` might be common enough that it doesn't warrant a custom typedef, but can't find anything on this in style guide, so it may be ok. Is there any other reason besides readability to use a typedef for this?

Reply:
No other reason, I found it easier when playing around with using a different type of structure. But I take your feedback and I'll change it to use the type directly.